JOSESwift/Sources/Algorithms.swift

```diff
@@ -54,32 +55,41 @@ public enum AsymmetricKeyAlgorithm: String, CaseIterable {
 /// - A256CBCHS512: [AES_256_CBC_HMAC_SHA_512](https://tools.ietf.org/html/rfc7518#section-5.2.5)
 public enum SymmetricKeyAlgorithm: String {
     case A256CBCHS512 = "A256CBC-HS512"
+    case A256CBCHS256 = "A256CBC-HS256"
```

Comment:
```suggestion
    case A128CBCHS256 = "A128CBC-HS256"
```

See [RFC-7518](https://tools.ietf.org/html/rfc7518#section-5). `A128CBC-HS512` means AES in CBC mode with a 128 bit encryption key and HMAC with SHA-256 with the HMAC output truncated to 128 bits as per [Section 5.2.3](https://tools.ietf.org/html/rfc7518#section-5.2.3).

Reply:
Done.